apps/meteor/client/views/admin/upgrade/UpgradePage/UpgradePage.tsx

```diff
@@ -95,11 +93,21 @@ const UpgradePage = (): ReactElement => {
 			{isMobile && <PageHeader title='' />}
 			{!hasConnection && <UpgradePageError />}
 			{hasConnection && isLoading && (
-				<Box pb='x24'>
+				<Box width={'100%'} height={'100%'} position={'absolute'} display={'flex'} justifyContent={'center'} alignItems={'center'}>
```

Comment:
It's safe to write string values for JSX element attributes without curly brackets:

```suggestion
				<Box width='100%' height='100%' position='absolute' display='flex' justifyContent='center' alignItems='center'>
```